airbyte-integrations/connectors/source-oracle-v2/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+micronaut:
+  application:
+    name: source-oracle-v2
+
+airbyte:
+  connector:
+    documentationUrl: "https://docs.airbyte.com/integrations/sources/oracle"
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 1 to 8]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Can we get this from the metadata file?  Do we need this application yaml as well?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/airbytehq/airbyte/37605?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
This is a great idea. Micronaut thinks the application.yml file is special but I imagine that we can tell it to look for properties in metadata.yaml instead. I'll file a ticket to investigate.

Reply:
#airbyte-internal-issues/7540